docs/en/writing-running-appium/caps.md

```diff
@@ -71,6 +71,7 @@ These Capabilities are available only on Android-based drivers (like
 |`androidInstallTimeout`|Timeout in milliseconds used to wait for an apk to install to the device. Defaults to `90000` |e.g., `90000`|
 |`androidInstallPath`| The name of the directory on the device in which the apk will be push before install. Defaults to `/data/local/tmp` |e.g. `/sdcard/Downloads/`|
 |`adbPort`|Port used to connect to the ADB server (default `5037`)|`5037`|
+|`systemPort` | `systemPort` used to connect to [appium-uiautomator2-driver](https://github.com/appium/appium-uiautomator2-driver), default `4724` | e.g., `4724` |
```

Comment:
I think it makes sense to also mention that this capability should be adjusted for each session running in parallel according to https://github.com/appium/appium/blob/55760b7b9e1c494c6a1f22d152f0bd1fe676dae7/docs/en/advanced-concepts/parallel-tests.md